app/views/lobby/home.scala

```diff
@@ -20,7 +20,7 @@ object home:
       moreJs = jsModuleInit(
         "lobby",
         Json
-          .obj("data" -> data, "i18n" -> i18nJsObject(i18nKeys))
+          .obj("data" -> data, "i18n" -> i18nJsObject(i18nKeys), "lastFeedRev" -> env.blog.dailyFeed.lastRev)
```

Comment:
should be in `data`

Reply:
Is it a good idea to add `blog` module as an `api` module dependency just for this? I suppose we could inject it into the data json within `lobby.home.apply` but there's a bit of fuckery for not much gain.

Reply:
nvm - i just saw `lazy val moduleCPDeps = moduleRefs map { sbt.ClasspathDependency(_, None) }` in build.sbt